src/query/test/fixtures/graph.ts

```diff
@@ -39,8 +39,7 @@ export const newNode =
     id: joinDepIDTuple(['registry', '', `${name}@${version}`]),
     name,
     version,
-    location:
-      'node_modules/.vlt/;;${name}@${version}/node_modules/${name}',
+    location: `node_modules/.vlt/;;${name}@${version}/node_modules/${name}`,
```

Comment:
Let's please use the `joinDepIDTuple` helper method from `@vltpkg/dep-id` whenever we're creating a new `DepID` in fixtures, here's an example on how to use it:

https://github.com/vltpkg/vltpkg/blob/395be19c5c99f7673de7c5512a25f5d070d0199a/src/graph/test/modifiers.ts#L50